pkg/volume/csi/csi_plugin.go

```diff
@@ -31,8 +31,7 @@ import (
 )
 
 const (
-	csiPluginName              = "kubernetes.io/csi"
-	csiVolAttribsAnnotationKey = "csi.volume.kubernetes.io/volume-attributes"
+	csiPluginName = "kubernetes.io/csi"
```

Comment:
@musse this should already be there, why adding it here?

Reply:
I only removed the spaces before the `=` sign.